examples/slice-pizza/datadog-app/package.json

```diff
@@ -3,16 +3,21 @@
     "version": "0.0.0",
     "private": true,
     "dependencies": {
-        "@datadog/ui-extensions-react": "0.30.1",
-        "@datadog/ui-extensions-sdk": "0.30.1",
+        "@chakra-ui/react": "^1.8.8",
+        "@datadog/ui-extensions-react": "0.31.0",
+        "@datadog/ui-extensions-sdk": "0.31.0",
+        "@emotion/react": "^11.9.0",
+        "@emotion/styled": "^11.8.1",
+        "@fontsource/bebas-neue": "^4.5.6",
+        "@fontsource/tourney": "^4.5.7",
         "@types/node": "^14.14.14",
         "@types/react": "^17.0.0",
         "@types/react-dom": "^17.0.0",
-        "milligram": "^1.4.1",
-        "react": "^17.0.1",
-        "react-dom": "^17.0.1",
+        "framer-motion": "^6.2.9",
+        "react": "^18.0.0",
+        "react-dom": "^18.0.0",
         "react-hook-form": "^7.28.1",
-        "react-scripts": "4.0.3",
+        "react-scripts": "5.0.0",
```

Comment:
Just updating these to what the rest of the repository is now using (and also what the latest version of Chakra needed).